scripts/build/ci-build.ps1

```diff
@@ -200,27 +204,50 @@ function Invoke-DotNetBuild() {
     Set-DotNetVersion
 
     $skippedAnalysis = $false
+    $leakPeriodVersion = Get-LeakPeriodVersion
+
     if ($isPullRequest) {
         Invoke-SonarBeginAnalysis `

```

Comment:
Make sure this is not called for external pull requests, where secure variables from travis aren't available.